src/components/charts/galaxySelector/Point.jsx

```diff
@@ -9,7 +9,7 @@ class Point extends React.PureComponent {
       {
         supernova: 20,
         galaxy: 40,
-      }[id] || 5
+      }[id] || 8
```

Comment:
Help me understand this return of `8` if the index requested does not exist...

Reply:
surely.  So, we've got this object literal `{ supernova: 20, galaxy: 40 }` and in the case of the GalaxySelector, as implemented in the Expanding Universe Investigation, id is either galaxy or supernova, since we like made that data ourselves, and we wanted a big circle around the galaxy and a small one around the supernova.  In the case of the HSC data, the ids are numbers, so because those keys don't exist in this object it'll return undefined which is falsey so it'll return the other side of the or.  It's very similar to the strategy we used for the QAs, blocks, and page layouts.

Reply:
I'm guessing the value of `8` holds significance to the response?

Reply:
20, 40, and 8 are the radii of points